sonic_platform_base/sfp_base.py

```diff
@@ -429,3 +432,20 @@ def get_error_description(self):
             like: "Bad EEPROM|Unsupported cable"
         """
         raise NotImplementedError
+
+    def reload(self):
```

Comment:
What does this API do?. Can you please change the docstring related to this API?.

Reply:
I will rename this to "refresh_xcvr_api" to make its purpose more clear. The idea is that each SfpBase derived object has an associated XcvrApi object appropriate for the type of transceiver module plugged in. At initialization or when a different module is plugged in the XcvrApi object would need to be refreshed.